alicloud/data_source_alicloud_rds_instances.go

```diff
@@ -0,0 +1,289 @@
+package alicloud
+
+import (
+	"log"
+	"regexp"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/requests"
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/rds"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceAlicloudRdsInstances() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAlicloudRdsInstancesRead,
+
+		Schema: map[string]*schema.Schema{
+			"name_regex": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"engine": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"db_instance_status": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"db_instance_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"instance_network_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
```

Comment:
instance_network_type can be removed and it can be clear when 'vpc_id' or 'vswitch_id' is be specified.

Reply:
done